### DISCUSSION TITLE
Repos stuck on Push/Publish

### DISCUSSION THREAD

**Discussion by StoutCrow** [community/community#59014] [Repositories]
### Select Topic Area

Bug

### Body

When i'm trying to push/publish this one repos, it gets stuck on "Pushing to Origin". The repos has 2 120mb files that are in LFS if that helps.

**Comment by ikozigbo** [comment]
If you're experiencing issues with pushing or publishing repositories on GitHub, there could be a few potential causes. Here are some steps you can take to troubleshoot and resolve the problem:

1. Check your internet connection: Ensure that you have a stable internet connection. You can try visiting other websites or services to verify if your internet connection is working properly.

2. Verify repository permissions: Make sure you have the necessary permissions to push or publish to the repository. If you're not the owner or a collaborator, you may need to request access or contact the repository owner to grant you the appropriate permissions.

3. Verify repository status: Sometimes, GitHub may experience service disruptions or maintenance periods that can temporarily impact repository operations. You can check the GitHub Status page (https://www.githubstatus.com/) to see if there are any ongoing issues.

4. Clear browser cache: Clearing your browser cache can help resolve caching-related issues that might affect repository operations. Try clearing your browser cache and then attempt to push or publish again.

5. Try a different Git client or IDE: If you're using a specific Git client or integrated development environment (IDE), try switching to a different one to see if the issue persists. This can help determine if the problem is specific to the client or IDE you're currently using.

6. Test with a different repository: Create a new, empty repository and try pushing or publishing to it. If you can successfully perform the operation on the new repository, it may indicate an issue with the specific repository you were previously working with.

7. Contact GitHub support: If none of the above steps resolve the issue, you can reach out to GitHub support for further assistance. They can investigate the problem and provide guidance tailored to your specific situation.

Remember, these are general troubleshooting steps, and the exact solution may depend on the specific details of the issue you're facing. Providing more specific information about the error message or any other relevant details would help in providing more targeted assistance.
